zcash-issue-dag.py

```diff
@@ -60,6 +72,8 @@
     'core': CORE_REPOS,
     'halo2': HALO2_REPOS,
     'wallet': WALLET_REPOS,
+    'ios': IOS_REPOS,
+    'android': ANDROID_REPOS,
```

Comment:
These are effectively sub-graphs, so let's make that clearer in the name:
```suggestion
    'wallet-ios': IOS_REPOS,
    'wallet-android': ANDROID_REPOS,
```